src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,346 @@
+        }
+        if (exitMenu(hotelName)) {
+            try {
+                serviceMenu();
+            } catch (IOException e) {
+                outputDataInConsole("Don't loaded Service menu" + e);
+            }
+        }
+        outputSplitLine();
+        List<Hotel> hotelList = hotelController.findHotelByName(hotelName);
+        if (hotelList.isEmpty()) {
+            outputDataInConsole("Hotel " + hotelName + " not found.");
+            findHotelByNameMenu();
+        } else {
+            outputDataInConsole("The list of found hotels by name: " + hotelName);
+            hotelList.forEach(item -> outputDataInConsole(item.toString()));
+        }
+        return hotelList;
+    }
+
+    private boolean exitMenu(String answerUser) {
+        return (answerUser.equals("0"));
+    }
+
+    private List<Hotel> findHotelByCityMenu() {
+        String cityName = null;
+        try {
+            cityName = String.valueOf(inputDataFromUser("Enter city name (to exit enter \"0\"): ",
+                    false, false, true, false));
+        } catch (IOException e) {
+            outputDataInConsole("Entered incorrect name of the city");
+        }
+        outputSplitLine();
+        List<Hotel> hotelList = hotelController.findHotelByCity(cityName);
+        if (exitMenu(cityName)) {
+            try {
+                serviceMenu();
+            } catch (IOException e) {
+                outputDataInConsole("Don't loaded Service menu" + e);
+            }
+        }
+        if (hotelList.isEmpty()) {
+            outputDataInConsole("City in " + cityName + " not found.");
+            findHotelByCityMenu();
+        } else {
+            outputDataInConsole("List of hotels in the city found : " + cityName);
+            hotelList.forEach(item -> outputDataInConsole(item.toString()));
+        }
+        hotelList.forEach(item -> outputDataInConsole(item.toString()));
+        return hotelList;
+    }
+
+    private Map<String, List<Room>> findRoomMenu() throws IOException {
+        String city = null;
+        Integer person = null;
+        Integer price = null;
+        Date reservedFrom = null;
+        Date reservedTo = null;
+        try {
+            city = String.valueOf(inputDataFromUser("Enter city name: ", false,
+                    false, true, false));
+            person = (Integer) inputDataFromUser("Enter the number of persons: ", true,
+                    false, false, false);
+            price = (Integer) inputDataFromUser("Enter maximum price rate: ", true,
+                    false, false, false);
+            reservedFrom = (Date) (inputDataFromUser("Enter arrival date in the format dd.mm.yyyy: ",
+                    false, false, false, true));
+            reservedTo = (Date) (inputDataFromUser("Enter the departure date in the format dd.mm.yyyy: ",
+                    false, false, false, true));
+            compareDate(reservedFrom, reservedTo);
+        } catch (IOException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)